src/providers/fault-count/cat-d/fault-count.cat-d.ts

```diff
@@ -288,7 +258,31 @@ export class FaultCountDHelper {
     return result;
   }
 
-  private static getSafetyQuestionsFaultCount = (
+  public static getVehicleChecksFaultCountCatD = (
```

Comment:
Convention is that public methods should be at the top of the file followed by protected and then private, can you move this function and the one below please.

Reply:
Thanks, will do